ckanext/who_afro/templates/sources/index.html

```diff
@@ -1,203 +1,216 @@
-{% extends "page.html" %} {% block subtitle %} {{ _('Data Sources') }}
-{%endblock subtitle%} {% block breadcrumb_content %}
-<li class="active">
-  {% link_for _('Data Sources'), named_route='sources.index' %}
-</li>
-{% endblock %} {% block content %}
-<div role="main" class="insights-page">
-  <div class="promoted-background">
-    {% snippet 'package/snippets/promoted_plus.html' %}
-    <div class="container">
-      <div class="promoted promoted-home col-xs-12 col-sm-10 col-md-8 col-lg-6">
-        <div class="promoted-container">
-          <p class="subtitle">{{ _('Home') }} /</p>
-          <h1 class="page-title">{{ _('Data Sources') }}</h1>
-          <p>
-            {{ _('The WHO African Regional Health Data Hub integrates public
-            health and related data from across the region and beyond. We draw
-            on a variety of sources including the following data sources.') }}
-          </p>
+{% extends "page.html" %}
+{% block subtitle %}
+    {{ _("Data Sources") }}
+{% endblock subtitle %}
+
+{% block breadcrumb_content %}
+    <li class="active">{% link_for _('Data Sources'), named_route='sources.index' %}</li>
+{% endblock %}
+
+{% block content %}
+    <div role="main" class="insights-page">
+        <div class="promoted-background">
+            {% snippet 'package/snippets/promoted_plus.html' %}
+            <div class="container">
+                <div class="promoted promoted-home col-xs-12 col-sm-10 col-md-8 col-lg-6">
+                    <div class="promoted-container">
+                        <p class="subtitle">{{ _("Home") }} /</p>
+                        <h1 class="page-title">{{ _("Data Sources") }}</h1>
+                        <p>
+                            {{ _('The WHO African Regional Health Data Hub integrates public
+                                  health and related data from across the region and beyond. We draw
+                                  on a variety of sources including the following data sources.') }}
+                        </p>
+                    </div>
+                </div>
+            </div>
+        </div>
+        <div class="container">
+            <ul class="icon-description-list">
+                <li>
+                    <div class="views-field views-field-icon">
```

Comment:
I'm wondering why inside the `<li>` elements we are not using bootstrap columns so that we don't need to set the widths ourselves. Of course, bootstrap will allow easily adapting to other devices.

Reply:
Honest answer - I copy pasted this from the countries page... which I copy pasted from somewhere else :rofl: because it was always meant to be a short fix. However, you are right - I will have a quick play

Reply:
@ChasNelson1990  I'm working on the countries list page and it is (was) a mess on mobile but looking a little better. Maybe we can reuse the code I'm writing?

Reply:
I have just pushed a new bootstrap 5 version for your review.

Reply:
[This](https://github.com/fjelltopp/ckanext-who-afro/commit/96714a292cff0d96dae4041c6e453a604d5b39ea) is the change I just did but on second thought reusing bootstrap's columns is probably still the best in your particular case.

Reply:
See my latest commit